packages/fluent-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -58,13 +74,23 @@ export default function CheckboxWidget<
   const _onFocus = ({ target: { value } }: FocusEvent<HTMLButtonElement>) => onFocus(id, value);
 
   const uiProps = _pick((options.props as object) || {}, allowedProps);
+  const description = options.description ?? schema.description;
 
   return (
     <>
+      {displayLabel && !!description && (
+        <DescriptionFieldTemplate
+          id={descriptionId<T>(id)}
+          description={description}
+          schema={schema}
+          uiSchema={uiSchema}
+          registry={registry}
+        />
+      )}
       <Checkbox
         id={id}
         name={id}
-        label={label || schema.title}
+        label={displayLabel ? label || schema.title : undefined}
```

Comment:
Honestly, at this point if `label` is an empty string then `schema.title` will be `undefined`, so do we really just change it to:
```suggestion
        label={displayLabel ? label || undefined : undefined}
```
And is there better boolean logic for the above condition to eliminate one of the `undefined`s?

Reply:
I can't think of a better way than your suggestion. I wonder if there's a case where rendering an empty string label is preferable to no label

Reply:
Maybe wrap the prop in a new utility 

```ts
function coerceEmptyStringToUndefined(str) {
  return str === "" ? undefined : str;
}
```

Reply:
how about?
```suggestion
        label={labelValue(label, displayLabel)}
```